core/controllers/base.py

```diff
@@ -143,9 +142,9 @@ def __init__(self, request, response):  # pylint: disable=super-init-not-called
         # Initializes the return dict for the handlers.
         self.values = {}
 
-        self.user = current_user_services.get_current_user()
+        self.current_user = current_user_services.get_current_user()
```

Comment:
I mean, I removed self.current_user variable and directly called current_user_services.get_current_user_id() at the places required.